test.js

```
@@ -4,3 +4,49 @@ import m from '.';
 test('empty test', t => {
 	t.is(m.thrown(), false);
 });
+
+test('out test', t => {
+	const out = m.throw('a', 'b', 'c');
+	t.is(out.body.code, 'InternalServer');
+	t.is(out.message, 'b');
+	t.is(out.body.errno, 'c');
+	t.is(out.context.debug[0], 'Invalid error type provided:InternalServerError');
+});
+
+test('error is present', t => {
```

Rename to
'should throw the correct error with a valid type'